.github/workflows/python.yml

```
@@ -0,0 +1,127 @@
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
+        with:
+          cache: true
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: |
+            ${{
+              hashFiles(format('src/{0}/.python-version', matrix.package)) != '' &&
```

actionlint (1.7.7)

undefined function "formatFile". available functions are "always", "cancelled", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)